polkadot-protocol/architecture/polkadot-chain/overview.md

```diff
@@ -1,4 +1,149 @@
 ---
 title: Overview
-description: TODO
----
\ No newline at end of file
+description: Explore Polkadot's core architecture, including its multi-chain vision, shared security, runtime upgrades, and the DOT token's governance and staking roles.
+---
+
+# Overview
+
+## Introduction
+
+Polkadot is a next-generation blockchain protocol designed to support a multi-chain future by enabling secure communication and interoperability between different blockchains. Built as a Layer-0 protocol, Polkadot introduces innovations like application-specific Layer-1 chains (parachains), shared security through Nominated Proof-of-Stake (NPoS), and seamless cross-chain interactions via its native Cross-Consensus Messaging Format (XCM). 
```

Comment:
```suggestion
Polkadot is a next-generation blockchain protocol designed to support a multi-chain future by enabling secure communication and interoperability between different blockchains. Built as a Layer-0 protocol, Polkadot introduces innovations like application-specific Layer-1 chains (parachains), shared security through Nominated Proof of Stake (NPoS), and seamless cross-chain interactions via its native Cross-Consensus Messaging Format (XCM). 
```

Below we don't use hyphens, and I think majority of the rest of the docs don't. We should probably check and make any updates to ensure we're being consistent